### ISSUE TITLE
Bagnon cata version 11.0.20  Missing ability to customize frame

### ISSUE THREAD

**Issue by Velarax** [Jaliborc/Bagnon#1967]
**Which software were you running?**
 - Addon version name:
 - Client used: [e.g. Vanilla 1.14.4, Retail PTR x.x.x, etc]
 
 Bagnon cata version 11.0.20

**Have you read the changelog?**
 Mark if read.  **YES**

**Please describe the bug.**

Bank and inventory frames are incomplete and frame sizing, slots, etc. options are unavailable.

Please refer to the screenshots I've included.

Also am includingt BugSack barf, which clearly indicates a bag brother framing issue.  Many thanks in advance.  
[bagnon bugz 924.txt](https://github.com/user-attachments/files/17117084/bagnon.bugz.924.txt)

![WoWScrnShot_092424_103020](https://github.com/user-attachments/assets/94630ba3-653e-47b0-9f78-8e58ca29ef1f)
![WoWScrnShot_092424_102955](https://github.com/user-attachments/assets/5d378d6f-9f7f-44a0-afbe-b7c849545b6f)

**Please describe how to reproduce it.**

**Any Screenshots?**
If applicable, please add screenshots to help explain your problem.

**Error Logs are Important!**
Type `/console scriptErrors 1` in the chat and reload the game. If an error window appears when reproducing the bug, please write here the first message (labelled 1/Many) that appeared. **:warning: By default, the last error message is shown, not the first!** You need to navigate back to the first error.

**Comment by Tupples** [comment]
After the blizzard update on Sept 24, the appearance settings are no longer available. I run 2 accounts. After updating all my addons, one account was fine with the current 11.0.20 version of bagnon, meaning my current appearance settings did not get reset, therefore I didn't notice any issue at the time. However my other account did get it's bagnon settings reset to where I need to resize my bags sizes. I am unable to do that because the appearance settings are now missing. Please put attention to this issue as it is servery impacting my ability to access bag content via bagnon. Thanks,

**Comment by Tupples** [comment]
The 11.0.21 release was made available on 9/28/24. This version did not resolve the appearance settings issue. The appearance settings are still missing, and we unable to resize inventory and bank windows. 

This has reduced the functionality of the addon from 'all-in-one' bag to '1 bag, 6 different tabs to view one at a time'
![bagnon appearance settings after blizz 1 15 4 56817 update](https://github.com/user-attachments/assets/559dd767-42a4-4ab0-a01e-4972d1d48315)
![bagnon appearance settings from youtube tutorial](https://github.com/user-attachments/assets/79ec0bf5-6fd3-47c2-9b6e-9d8938b34fa8)

**Comment by Lowdpska** [comment]
a similar problem

**Comment by Tupples** [comment]
I'm happy to say that I was able to resolve this issue myself. On 10/3 Blizz released 1.15.4.56857 for Classic_Era. I have 2 accounts that I play thgourh on the same PC, sharing the same Addon folder. As stated previously, one account Bagnon worked on, the other did not. Today after the new blizz update, I found that the account that bagnon worked also started showing me the appearance settings again. However on my non functional bagnon account, the condition was still the same. I then started experimenting with reset the profiles in the WTF folder. 

Bottom line is, by removing the Bagnon.lua file from C:\Program Files\World of Warcraft\_classic_era_\WTF\Account\(account name)\SavedVariables ... that allowed my bagnon to work. When I logged off, a new bagnon.lua was created and it continues to function.
